lib/kapost/bootstrapper.rb

```diff
@@ -91,9 +91,18 @@ def installed?(command)
 
     def right_version?(command, expected_version)
       version, status = cli.capture2e "#{command} --version"
-      status.success? && version.include?(expected_version) && version
+      if expected_version[0] == '^'
+        next_major = (expected_version[1].to_i + 1).to_s
+        Gem::Version.new(version) >= Gem::Version.new(expected_version[1..-1]) && Gem::Version.new(version) < Gem::Version.new(next_major)
+      elsif expected_version[0] == "="
+        Gem::Version.new(version) == Gem::Version.new(expected_version[1..-1])
+      else
+        local_version = Semantic::Version.new(version)
```

Comment:
this is the greater than/less than condition? Can't `Gem::Version.new(version)` handle this too?

Reply:
Semver prefixes supported by `Semantic::Version.new(version)`:
`>`, `>=`, `<`, `<=`, and no prefix

Things it can't handle: